AppCenterData/AppCenterDataTests/MSDataTests.m

```diff
@@ -1114,7 +1114,7 @@ - (void)testCreateWithPartitionGoldenPath {
                                  XCTAssertTrue(completionHandlerCalled);
                                  XCTAssertNotNil(actualDocumentWrapper.deserializedValue);
                                  XCTAssertTrue([[actualDocumentWrapper documentId] isEqualToString:@"standalonedocument1"]);
-                                 XCTAssertTrue([[actualDocumentWrapper partition] isEqualToString:@"readonly"]);
+                                 XCTAssertTrue([[actualDocumentWrapper partition] isEqualToString:@"user-123"]);
```

Comment:
You don't need a custom prefixed user partition it's only used for the user table in the storage. These calls are more likely to use `kMSDataUserDocumentsPartition`.

Reply:
I was wrong the returned metadata from the document wrapper should contain the user-account.